src/ol/PluggableMap.js

```diff
@@ -464,6 +465,10 @@ class PluggableMap extends BaseObject {
 
   }
 
+  /**
+   * @abstract
```

Comment:
I think TypeScript does not support the `@abstract` annotation, so it can be removed.